readme.md

```diff
@@ -67,6 +68,7 @@ $ np --help
     --no-tests          Skips tests
     --yolo              Skips cleanup and testing
     --no-publish        Skips publishing
+    --preview           Show only tasks that will be executed
```

Comment:
This is not the correct way to phrase it.

```suggestion
    --preview           Show tasks without actually executing them
```

Reply:
If updated, remind to update the 2 other similar places.